libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -289,6 +295,15 @@ class NavigationOptions private constructor(
         fun eventsAppMetadata(eventsAppMetadata: EventsAppMetadata?): Builder =
             apply { this.eventsAppMetadata = eventsAppMetadata }
 
+        /**
+         * Enables analyzing data from sensors for better location prediction in case of a weak
+         *  GPS signal, for example in tunnel.
+         * Warning: don't enable sensors if you emulate location updates. The SDK ignores
+         *  location updates which don't match data from sensors.
```

Comment:
@SiarheiFedartsou , does this description make sense?

Reply:
I think so.

Reply:
Should we explicitly mention that this has an impact on battery life? Has this been measured @SiarheiFedartsou?